test/resourceNameTest.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Story = require('../model/constructor.js');
+const superagent = require('superagent');
+const storage = require('../lib/storage.js');
+const apiURL = 'http://localhost:3000';
+
+describe('testing constructor model function', function (){
```

Comment:
Modularize your constructor tests away from your router tests. I.e. put your construction function test in its own separate file and your routers will get tested in their own file.